docopt.cpp

```diff
@@ -70,20 +70,10 @@ class Tokens {
 
 	static Tokens from_pattern(std::string const& source) {
 		static const std::regex re_separators {
-			"(?:\\s*)" // any spaces (non-matching subgroup)
-			"("
-			"[\\[\\]\\(\\)\\|]" // one character of brackets or parens or pipe character
-			"|"
-			"\\.\\.\\."  // elipsis
-			")" };
```

Comment:
I'm not sure I like this change ... I found these comments helpful as documentation on what the regex means.